logstash-core/spec/support/mocks_classes.rb

```diff
@@ -1,79 +1,91 @@
 # encoding: utf-8
```

Comment:
Not a lot changed here, only the addition of a `DummyInput`, I've just added a new module and I had to reformat the other code.